## 62% of end-end tests have passed
```
Total executed tests: 8
Failed tests: 2
Expected to fail tests: 1
Passed tests: 5
```
```
IDs of failed tests: 702745,703133 
```
```
IDs of expected to fail tests: 702843 
```
<h3>Failed tests (2)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=fix/hide-mercuryo&PR_ID=21626&APK_NAME=21626.apk&TR_CASE_IDS=702745,703133">Rerun failed tests</a></li><br/><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3725918">test_restore_multiaccount_with_waku_backup_remove_profile_switch</a>, id: 703133 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `accessibility id`: `show-profiles` </div><div>Device 1: Tap on found: Button </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:245: in test_restore_multiaccount_with_waku_backup_remove_profile_switch
    self.errors.verify_no_errors()
base_test_case.py:192: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 zQ3...dWXh5 was not restored as a contact from waku backup!
E    zQ3...Vacac was not restored as a contact from waku backup!
E    admin_open was not restored from waku-backup!!
E    member_open was not restored from waku-backup!!
E    admin_closed was not restored from waku-backup!!
E    member_closed was not restored from waku-backup!!
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1296bcab500b408fa72aa8c4df340d1d?auth=35e8352cb934ffaa9e56ebedecef2278#86">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3725916">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><p><blockquote><div>Device 2: Button element not found </div><div>Device 2: Wait for element `Button` for max 30s and click when it is available </div></blockquote></p>

```
Test setup failed: critical/chats/test_1_1_public_chats.py:24: in prepare_devices
    self.loop.run_until_complete(run_in_parallel(((self.device_1.create_user, {'enable_notifications': True,
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:52: in run_in_parallel
    returns.append(await k)
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../views/sign_in_view.py:249: in create_user
    self.not_now_button.wait_and_click()
../views/base_element.py:100: in wait_and_click
    self.wait_for_visibility_of_element(sec)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[@text='Not now']` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6bdf725dd0744b2aa232419232279f30?auth=db9273be04901e13db2214e74d77947c">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d9061122bc7f4ebcaf00e82f2b41cf7f?auth=9f593fe7876d590cbf5a054ba5fe61c3">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Expected to fail tests (1)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3725917">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td>

```
Test is not run, e2e blocker  
```

<b>[[reason: [NOTRUN] Skipped due to waku issue on staging fleet]]</b><br/><br/></td></tr></tbody></table></details><h3>Passed tests (5)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3725920">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3725919">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3725921">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3725922">test_wallet_balance_mainnet</a>, id: 740490 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3725915">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1296bcab500b408fa72aa8c4df340d1d?auth=35e8352cb934ffaa9e56ebedecef2278#54">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/src/status_im/contexts/wallet/sheets/buy_token/view.cljs
+++ b/src/status_im/contexts/wallet/sheets/buy_token/view.cljs
@@ -49,6 +49,7 @@
                   (rf/dispatch [:wallet/get-crypto-on-ramps])))
   (let [crypto-on-ramps                 (rf/sub [:wallet/crypto-on-ramps])
         account                         (rf/sub [:wallet/current-viewing-account-or-default])
+        has-recurrent?                  (seq (:recurrent crypto-on-ramps))
         [selected-tab set-selected-tab] (rn/use-state initial-tab)
         [min-height set-min-height]     (rn/use-state 0)
         on-layout                       (rn/use-callback
@@ -56,18 +57,27 @@
                                            (oops/oget % :nativeEvent :layout :height)))]
     [:<>
      [quo/drawer-top {:title (or title (i18n/label :t/ways-to-buy-assets))}]
-     [quo/segmented-control
-      {:size            32
-       :container-style style/tabs
-       :default-active  initial-tab
-       :on-change       set-selected-tab
-       :data            tabs}]
-     [rn/flat-list
-      {:data        (if (= selected-tab :recurrent)
-                      (:recurrent crypto-on-ramps)
-                      (:one-time crypto-on-ramps))
-       :on-layout   on-layout
-       :style       (style/list-container min-height)
-       :render-data {:tab     selected-tab
-                     :account account}
-       :render-fn   crypto-on-ramp-item}]]))
+     (when has-recurrent?
+       [:<>
+        [quo/segmented-control
+         {:size            32
+          :container-style style/tabs
+          :default-active  initial-tab
+          :on-change       set-selected-tab
+          :data            tabs}]
+        [rn/flat-list
+         {:data        (if (= selected-tab :recurrent)
+                         (:recurrent crypto-on-ramps)
+                         (:one-time crypto-on-ramps))
+          :on-layout   on-layout
+          :style       (style/list-container min-height)
+          :render-data {:tab     selected-tab
+                        :account account}
+          :render-fn   crypto-on-ramp-item}]])
+     (when-not has-recurrent?
+       [rn/flat-list
+        {:data        (:one-time crypto-on-ramps)
+         :on-layout   on-layout
+         :style       (style/list-container min-height)
+         :render-data {:account account}
+         :render-fn   crypto-on-ramp-item}])]))

--- a/status-go-version.json
+++ b/status-go-version.json
@@ -3,7 +3,7 @@
     "_comment": "Instead use: scripts/update-status-go.sh <rev>",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "v3.10.0",
-    "commit-sha1": "906c50bf26ef85b9a15eb3b640a6cbe2e1071260",
-    "src-sha256": "05di85jgy2avnl6y1pgx1kzqng9k068braj5fhaknpalm8yyln7k"
+    "version": "fix/hide-mercuryo-provider",
+    "commit-sha1": "2d4174571d1d536ac88d5756b90d4864df120336",
+    "src-sha256": "1j6l1s3dqr8qmh6ikh26n9bfx8cxnp75fg0ljx7k1x4yv9r4h68x"
 }

--- a/status-go-version.json
+++ b/status-go-version.json
@@ -4,6 +4,6 @@
     "owner": "status-im",
     "repo": "status-go",
     "version": "fix/hide-mercuryo-provider",
-    "commit-sha1": "2d4174571d1d536ac88d5756b90d4864df120336",
-    "src-sha256": "1j6l1s3dqr8qmh6ikh26n9bfx8cxnp75fg0ljx7k1x4yv9r4h68x"
+    "commit-sha1": "43056094932b9b15cce5827a19dafc27cf8160a6",
+    "src-sha256": "08hdi66wq9zxkih2a84526a96qzq9fsnq4p8qdf9bk340y1sgsrj"
 }

